numba/cuda/tests/cudadrv/test_emm_plugins.py

```diff
@@ -0,0 +1,188 @@
+import ctypes
+import numpy as np
+import sys
+import weakref
+
+from numba import cuda
+from numba.cuda.testing import unittest, SerialMixin, skip_on_cudasim
+
+not_linux = not sys.platform.startswith('linux')
```

Comment:
Suggest removing this and altering the test that makes use of it to use `@numba.tests.support.linux_only`.

Reply:
Will do - I copied this from `test_ipc` so I'll fix it there too.

Reply:
I just realised that `linux_only` is the opposite of `not_linux`, so I'll add `not_linux` to `numba.tests.support` instead.

Reply:
I've got confused by the double negative of skipping on not linux. Never mind!

Reply:
Now uses `@linux_only`.